test/tools/libtesteth/Options.cpp

```diff
@@ -279,6 +279,19 @@ Options::Options(int argc, char** argv)
 		}
 	}
 
+	//check restrickted options
+	if (createRandomTest)
+	{
+		string optionlist = "trValueIndex, trGasIndex, trDataIndex, nonetwork, singleTest, performance, quadratic, memory, inputLimits, bigData, wallet, stats, filltests, fillchain";
+		if (trValueIndex >= 0 || trGasIndex >= 0 || trDataIndex >= 0 || nonetwork || singleTest
+			|| 	performance || quadratic || memory || inputLimits || bigData || wallet
+			|| 	stats || filltests || fillchain)
+		{
+			cerr << "--createRandomTest could not be used with one of the options: " + optionlist << endl;
```

Comment:
Also I don't see why you need `optionlist` in a variable, just output it here as a literal